include/ocpp/v201/smart_charging.hpp

```diff
@@ -27,12 +28,19 @@ enum class ProfileValidationResultEnum {
 /// to calculate the composite schedules
 class SmartChargingHandler {
 private:
+    std::map<int32_t, std::unique_ptr<Evse>>& evses;
```

Comment:
Here's another of these static code analysis checkers

Reply:
This is one of the area where the checker that codacy uses seems to consistently have problems. The member is clearly used in the .cpp but a false positive is raised in the header anyway